### ISSUE TITLE
Merge 'ATP hydrolysis coupled transmembrane transport' into parent

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#17374]
Hello, 

The 'ATP hydrolysis' part of this term is redundant with the MF. I propose to merge as follows: 

Term | Merge into
-- | --
GO:0090662 ATP hydrolysis coupled transmembrane transport'  | GO:0055085 transmembrane transport
GO:0099131 ATP hydrolysis coupled ion transmembrane transport'  | GO:0034220 ion transmembrane transport
'GO:0015991 ATP hydrolysis coupled proton transport'  |GO:1902600 proton transmembrane transport
GO:0099133 ATP hydrolysis coupled anion transmembrane transport |  GO:0098656 anion transmembrane transport
GO:0099132 ATP hydrolysis coupled cation transmembrane transport | GO:0098655 cation transmembrane transport

Thanks, Pascale

**Comment by deustp01** [comment]
But the merge loses information. Per the textbook, distinct molecular mechanisms / functions mediate the energy-dependent (mostly, maybe always ATP-dependent) transport and energy-independent transport (aka facilitated diffusion). In the first case, the transport protein either binds ATP or is phosphorylated and as a result undergoes a conformational change so that a bound molecule or ion is moved across the membrane and released on the other side. Regenerating the transporter for a new round of transport involves either hydrolysis of the bound phosphate or conversion of the bound ATP to ADP which is released. In both cases, the net result is movement of a physical entity from [here] to [there] accompanied by the conversion of ATP top ADP. In the facilitated diffusion case the transporter forms a pore whose three-dimensional shape accommodates only specific molecules which then traverse the pore down a concentration gradient. 

Bottom line (my opinion): classification by mechanism is a useful differentium orthogonal to the classification by chemical class of entity transported already present in children of GO:0055085 transmembrane transport. Biologists put a lot of energy (sorry) into distinguishing transport involving pumps from transport by diffusion, so terms are needed to allow that distinction to be annotated. So this whole argument goes away if I'm overlooking other GO terms and annotation strategies that, especiaslly with GO-CAM available, allows this distinction to be made.

**Comment by ValWood** [comment]
@deustp01 this distinction is mechanistic and fits better into the MF ontology than biological process. DO you think we need it in both?

Although I strongly believe we need transmembrane transport in process, I think this distinction could be MF only?

**Comment by deustp01** [comment]
OK. I missed the process - function distinction above. If the GO convention is that anything that gets you from "A" to "B" is an example of the same process, regardless of the molecular mechanisms involved (which sounds OK to me, just reciting because I haven't thought this through before), then one transport process term can encompass energy-dependent and energy-independent functions.

**Comment by pgaudet** [comment]
Great, thanks for the feedback. I'll go ahead with the merge.
